.github/workflows/lgtm.yaml

```diff
@@ -0,0 +1,16 @@
+name: "Merge on lgtm label"
```

Comment:
I'm assuming the label will take effect when folks with the right permissions add it, right?

Reply:
Right, only those who have sufficient permissions will take effect, also we have branch protection rules enabled so PR's won't get merged unless all the checks are passed.